lib/ioc-rc

```diff
@@ -543,10 +543,12 @@ __rc_jails () {
         local name="$(zfs get -H -o value org.freebsd.iocage:host_hostuuid \
                     $jail)"
         local boot="$(zfs get -H -o value org.freebsd.iocage:boot $jail)"
+        local _jail_hostid="$(zfs get -H -o value org.freebsd.iocage:hostid \
+                        $jail)"
         local priority="$(zfs get -H -o value org.freebsd.iocage:priority \
                         $jail)"
 
-        if [ "$boot" == "on" ] ; then
+        if [ "$boot" == "on" ] && [ "$_jail_hostid" == "$hostid" ] ; then
```

Comment:
As you can see github and programming are not my speciallity!
But I will try it !



Op 14/09/15 om 23:46 schreef Brandon Schneider:
>
> In lib/ioc-rc
> <https://github.com/iocage/iocage/pull/222#discussion_r39451596>:
>
> >          local priority="$(zfs get -H -o value org.freebsd.iocage:priority \
> >                          $jail)"
> >  
> > -        if [ "$boot" == "on" ] ; then
> > +        if [ "$boot" == "on" ] && [ "$_jail_hostid" == "$hostid" ] ; then
>
> I've been trying to convert all of our if statements to '-a' or '-o'.
> Can you replace this with |if [ "$boot" == "on" -a "$_jail_hostid" ==
> "$hostid" ] ; then| please?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iocage/iocage/pull/222/files#r39451596>.
>